plugins/modules/gpg_keypair.py

```diff
@@ -0,0 +1,507 @@
+        description:
+            - List of subkeys with their own respective key types, lengths, curves, and usages.
+        type: list
+        elements: dict
+        default: []
+        suboptions:
+            subkey_type:
+                description:
+                 - Similar to O(key_type).
+                 - Also supports ECDH and ELG keys.
+                type: str
+                choices: [ 'RSA', 'DSA', 'ECDSA', 'EDDSA', 'ECDH', 'ELG' ]
+            subkey_length:
+                description:
+                    - Similar to O(key_length).
+                    - For ELG subkeys, the minimum length is V(1024) bits, the maximum length is V(4096) bits, and the default length is V(3072) bits.
+                type: int
+            subkey_curve:
+                description:
+                    - Similar to O(key_curve).
+                    - V(cv25519) is supported if subkey_type is V(ECDH).
+                    - This is required if subkey_type is V(ECDSA), V(EDDSA), or V(ECDH) and it is ignored if subkey_type is V(RSA), V(DSA), or V(ELG).
+                type: str
+                choices: ['nistp256', 'nistp384', 'nistp521', 'brainpoolP256r1', 'brainpoolP384r1', 'brainpoolP512r1', 'secp256k1', 'ed25519', 'cv25519']
+            subkey_usage:
+                description:
+                    - Similar to O(key_usage).
+                    - V(encr) is supported if subkey_type is V(RSA), V(ECDH), or V(ELG).
+                    - If subkey_type is V(ECDH) or V(ELG), only V(encr) is supported.
+                type: list
+                elements: str
+                default: []
+                choices: [ 'encr', 'sign', 'auth' ]
+    expire_date:
+        description:
+            - Sets the expire date for the key.
+            - If O(expire_date=0), the key never expires.
+            - If O(expire_date=<n>), the key expires in V(n) days.
+            - If O(expire_date=<n>w), the key expires in V(n) weeks.
+            - If O(expire_date=<n>m), the key expires in V(n) months.
+            - If O(expire_date=<n>y), the key expires in V(n) years.
+            - If left unspecified, any created GPG keys never expire.
+        type: str
+    name:
+        description:
+            - Specifies a name for the key's user id.
+        type: str
+    comment:
+        description:
+            - Specifies a comment for the key's user id.
+        type: str
+    email:
+        description:
+            - Specifies an email for the key's user id.
+        type: str
+    passphrase:
+        description:
+            - Passphrase used to decrypt an existing private key or encr a newly generated private key.
+        type: str
+    fingerprints:
+        description:
+            - Specifies keys to match against.
+        type: list
+        elements: str
+        default: []
+    force:
+        description:
+            - If O(force=True), key generation is executed using the module's options, even a matching key is found.
```

Comment:
```suggestion
            - If O(force=true), key generation is executed using the module's options, even a matching key is found.
```
Always use `true` and `false` for booleans outside Python code. (Documentation is not considered Python code, since it's YAML.)